client/src/App.jsx

```diff
@@ -42,24 +37,20 @@ export default function App() {
         throw new Error('Response is not in JSON format');
       }
       const data = await response.json();
-     
+      // Set recommendations state if needed
     } catch (error) {
-     
+      // Handle error
     }
   };
-  
-
-
-
 
-  
   useEffect(() => {
     // Fetch recommendations when the component mounts
     // fetchRecommendations();
   }, []);
 
   return (
     <>
+      <NavBar /> {/* Navbar global platzieren */}
```

Comment:
added NavBar into it, so it will be globally. The reason why i did this is applying the DRY Ruleset (Dont Repeat Yourself). instead of including the NavBar in every single file, i added it once and it works for every Route/Page you are going